src/components/getAnnualResultsByOption.js

```diff
@@ -96,24 +81,73 @@ export function getAnnualResultsByOption ({
   return annualResultsByOption
 }
 
-function getAnnualResult (budget, pmt, mortgageRate, investmentRate, result) {
-  return {
-    pmt,
+function getAnnualResult (
+  mortgageRate,
+  investmentRate,
+  inflationRate,
+  lastResult
+) {
+  let pmt = lastResult.pmt
+
+  if (
+    lastResult.loanAmount <
+    FV(mortgageRate / COMPOUND_FREQUENCY, COMPOUND_FREQUENCY, pmt, 0)
+  ) {
+    pmt = PMT(
+      mortgageRate / COMPOUND_FREQUENCY,
+      COMPOUND_FREQUENCY,
+      lastResult.loanAmount
+    )
+  }
+
+  console.log(
+    'investmentPMT',
+    FV(
+      investmentRate / COMPOUND_FREQUENCY,
+      COMPOUND_FREQUENCY,
+      lastResult.budget - pmt,
+      0
+    )
+  )
+
+  const resultBeforeInflation = {
+    budget: lastResult.budget,
+    pmt: pmt,
     loanAmount:
       -1 *
       FV(
         mortgageRate / COMPOUND_FREQUENCY,
         COMPOUND_FREQUENCY,
         pmt,
-        result.loanAmount
+        lastResult.loanAmount
       ),
     investmentAmount:
       -1 *
       FV(
         investmentRate / COMPOUND_FREQUENCY,
         COMPOUND_FREQUENCY,
-        budget - pmt,
-        result.investmentAmount
+        lastResult.budget - pmt,
+        lastResult.investmentAmount
       )
   }
+
+  const resultAfterInflation = {}
+
+  _.forOwn(
+    resultBeforeInflation,
+    (value, key) =>
+      (resultAfterInflation[key] = getAnnualValueAfterInflation(
+        value,
+        inflationRate
+      ))
+  )
```

Comment:
@jgozal I don't think this math is actually right. It's not matching up what the other guy had. Will need to dig further, b/c I'm struggling to get inflation to work correctly w/ each time step

Reply:
no worries @hdavidzhu! it's definitely not easy

Reply:
no worries @hdavidzhu  I'm sure it's not as easy as we thought

Reply:
@hdavidzhu please let me know when this PR is ready for review.

Reply:
Go for it!

Reply:
nice!! I love how it looks, will review the code now